arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts

```diff
@@ -24,8 +24,6 @@
 #define spi6 _spi6
 #define uart0 _uart0
```

Comment:
This commit could go before the addition of the bcm2712d0 variant, to avoid modifying it.